kpi/deployment_backends/openrosa_backend.py

```diff
@@ -873,13 +874,14 @@ def rename_enketo_id_key(self, previous_owner_username: str):
         parsed_url = urlparse(settings.KOBOCAT_URL)
         domain_name = parsed_url.netloc
         asset_uid = self.asset.uid
-        enketo_redis_client = get_redis_connection('enketo_redis_main')
-
         try:
+            enketo_redis_client = get_redis_connection('enketo_redis_main')
             enketo_redis_client.rename(
                 src=f'or:{domain_name}/{previous_owner_username},{asset_uid}',
                 dst=f'or:{domain_name}/{self.asset.owner.username},{asset_uid}'
             )
+        except InvalidCacheBackendError:
+            pass
```

Comment:
Why do we need this? We really don't want to pass if `enketo_redis_main` does not exist. It has to raise a 500.

Reply:
Please leave a comment there to explain why we need that.